src/store/zoom.ts

```diff
@@ -0,0 +1,151 @@
+import { createSlice, PayloadAction, createSelector } from '@reduxjs/toolkit';
+import { RootState } from '../store';
+
+interface ZoomState {
+	byId: {
+		[zoomId: string]: {
+			zoomId: string;
+			discordId?: string;
+			name: string;
+		};
+	};
+	online: string[];
+	temporary: string[];
+	hasSeenStart: boolean;
+	active: boolean;
+	joinUrl: string;
+	lastZoomTime: number;
+}
```

Comment:
Can you add JSDoc `/** ... */` comments explaining what each of these things is? (Top level keys at least, I think the properties of byId are pretty self explanatory)